docker/README.md

```diff
@@ -95,18 +96,18 @@ docker build -f Dockerfile-dev -t simtools-dev .
 
 Use the docker container in the same way as above, replacing `ghcr.io/gammasim/simtools-dev:latest` by `simtools-dev`.
 
-## Container for CORSIKA and sim\_telarray
+## Container for CORSIKA and simtelarray
 
 Provide a container including the following the CORSIKA and sim\_telarray simulation software packages.
 
 ### Download from simtools container repository
 
-Packages are available from the [simtools container repository](https://github.com/gammasim/containers/pkgs/container/simtools-simtel).
+Packages are available from the [simtools container repository](https://github.com/orgs/gammasim/packages?repo_name=simtools) for different options (e.g., sim\_telarray versions, hadronic interaction models, CTA MC productions).
 
-To download and run a prepared container in bash:
+To download and run a prepared container in bash using e.g., a container for prod6:
 
 ```bash
-docker run --rm -it -v "$(pwd)/external:/workdir/external" ghcr.io/gammasim/simtools-simtelarray:latest bash
+docker run --rm -it -v "$(pwd)/external:/workdir/external" ghcr.io/gammasim/simtools-corsika-sim-telarray-qgs2-prod6-baseline-240318:latest bash
 ```
 
 ## Build a new container locally
```

Comment:
I would additionally include here how to build containers locally with other options, e.g. `docker build -f Dockerfile-simtelarray --build-arg="BUILD_OPT=prod5" -t simtel-docker-dev .` or `docker build -f Dockerfile-simtelarray --build-arg="HADRONIC_MODEL=sibyll" -t simtel-docker-dev .`. I tested locally and at least these two examples work (the option passed in command line indeed overwrites whatever is in the docker file, which is good).

Reply:
OK, added (this is standard docker and the github workflow shows that it is working nicely)